src/main/java/amazon/AllSearchResultPage.java

```diff
@@ -0,0 +1,55 @@
+package amazon;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+import java.util.List;
+
+public class AllSearchResultPage extends BasePage {
+
+    public AllSearchResultPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(xpath = "//*[text()='Best Seller']")
+    protected List<WebElement> listItemWhereWriteBestSeller;
+    @FindBy(xpath = "//*[text()='Next']")
```

Comment:
Change the selector. The easiest way to detect the web element doesn't mean the most reliable way.
You can have multiple "Next" texts in your page in the future, more than one.
It's easily can be replaced with more reliable selector, Fix it.